pkg/component/tls.go

```diff
@@ -47,9 +47,7 @@ func WithTLSClientAuth(auth tls.ClientAuthType, cas *x509.CertPool, verify func(
 func (c *Component) GetTLSConfig(ctx context.Context, opts ...TLSConfigOption) (conf *tls.Config, err error) {
 	if c.acme != nil {
 		conf = &tls.Config{
-			GetCertificate:           c.acme.GetCertificate,
-			PreferServerCipherSuites: true,
-			MinVersion:               tls.VersionTLS12,
+			GetCertificate: c.acme.GetCertificate,
 			NextProtos: []string{
 				"h2", "http/1.1",
```

Comment:
We should make sure that we also have `h2` and `http/1.1` in the "normal" tls config below